README.md

```diff
@@ -11,189 +12,67 @@ This project explores the use of large language models (LLMs) for product recomm
-4. [Installation](#installation)
-5. [Usage](#usage)
-6. [Training the Model](#training-the-model)
-7. [Evaluation](#evaluation)
-8. [Contributing](#contributing)
-9. [License](#license)
-
----
-
-## **Data**
-
-This project uses Amazon Appliance Reviews data for training, validation, and testing. The dataset can be broadky categorised into:
-- **User Reviews**: Including `user_id`, `review_text`, `parent_asin`, and `timestamp`.
-- **Product Metadata**: Including `parent_asin`, `title`, `category`, `features`, `price`, and more.
-
-Each row in the dataset is processed to generate the next 10 items that the user interacted with. Data is split into training, validation, and test sets, keeping the temporal nature of user interactions intact.
-
-[Download data here](https://amazon-reviews-2023.github.io/)
-
----
-
-## **Model Architecture**
-
-The project uses the GPT-2 medium architecture, consisting of:
-- **Embedding Dimension**: 1024
-- **Number of Layers**: 24
-- **Number of Attention Heads**: 16
-
-The model is fine-tuned to take as input:
-- **User Reviews**: User-written text.
-- **Item Metadata**: Key features like title, category, and other details.
-
-### **Loss Function**:
-We calculate the loss using cross-entropy between the predicted next items and the actual items the user interacted with.
-
----
-
-## **Installation**
-
-To run this project locally, follow these steps:
-
-1. **Clone the Repository**:
-    ```bash
-    git clone https://github.com/yourusername/recsys-llm.git
-    cd recsys-llm
-    ```
-
-2. **Install Dependencies**:
-    ```bash
-    pip install -r requirements.txt
-    ```
-
-3. **Set Up the Data**:
-    - Download the data `Appliances.jsonl` and `meta_Appliances.jsonl` [data here](https://amazon-reviews-2023.github.io/)
-    - Place the data in `data/raw/` folder
-
-4. **Configure Environment Variables**:
-    If using CUDA, ensure your environment is properly set up for PyTorch GPU usage.
-
-
-To run the script. Go to the command line and run the following:
-
-If you want to run the model using Alpaca-style prompt **without** LORA (which is our baseline model). Run 
-```bash
-python gpt-experiment.py --run_solution baseline
-```
-
-Use Alpaca-style prompt **with** LORA
-```bash
-python gpt-experiment.py --run_solution alpaca_and_lora
-```
-
-If you want to run the model using Phi3-style prompt **without** LORA
-```bash
-python gpt-experiment.py --run_solution phi3_prompt
-```
-
-If you want to run the model using Phi3-style prompt **with** LORA
-```bash
-python gpt-experiment.py --run_solution phi3_and_lora
-```
-
-
----
-
-## **Usage**
-
-### **Data Preprocessing**
-
-To curate quality training data, we focus on users that has purchased at least 10 materials.
-The data should be preprocessed before training. The `get_next_10_items` function processes user interaction data and ensures that each entry contains a list of the next 10 items based on historical interactions.
+## **Evaluation**
```

Comment:
Thanks for your contribution.
However, it seems you deleted a huge chunk of the README file. Could you please check this again?